pages/tutorials/migrating_from_bamboo.md.erb

```diff
@@ -30,7 +30,7 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
-The `block` step will also wait for previous steps to complete but instead of automatically continuing, it will stop the build and require a user to manually "Unblock" the Pipeline by clicking the "Continue" button in the web interface, or use the [Unblock Job](/docs/api/jobs#unblock-a-job) REST API endpoint. These are the equivalent of a *Manual Stage* in Bamboo.
+The `block` step will also wait for previous steps to complete. Still, instead of automatically continuing, it will stop the build and require a user to manually "unblock" the pipeline by clicking the "Continue" button in the Buildkite UI, or use the [Unblock Job](/docs/api/jobs#unblock-a-job) REST API endpoint. These are the equivalent of a *Manual Stage* in Bamboo.
```

Comment:
I found this one super confusing as is, 


```suggestion
Instead of the `wait` step above, you could use a `block` step to stop the build **and** require a user to manually "unblock" the pipeline by clicking the "Continue" button in the Buildkite UI, or use the [Unblock Job](/docs/api/jobs#unblock-a-job) REST API endpoint. This is the equivalent of a *Manual Stage* in Bamboo.
```